plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/MySQLDialect.java

```diff
@@ -270,7 +270,7 @@ protected String escapeString(@NotNull String string, @Nullable String[] quotes)
         if (quotes != null) {
             string = string.replace(quotes[0], quotes[0] + quotes[0]);
         } else {
-            string = string.replace("'", "''").replace("`", "``");
+            return super.escapeString(string);
```

Comment:
I'm sure this was done for some reason and I don't like the idea of just removing it.

Reply:
I agree, it might have been put for a reason and from git blame I can see that you added it? 😅 

Either way the reason I thought it should be removed it that I checked Sqlite and Postgresql don't use this escaping so it might have been added for a reason but I believe this wasn't the correct way of fixing it so maybe remove it now and let me know the reason and I will try to fix the other problem without having to compromise the export feature

Reply:
Here is the initial issue about these escaping #17619 and the following fix was done #17870
I checked that your fix doesn't broke that issue.

So, maybe we can rely on my investigation and merge this PR.

Reply:
Also the build of this PR passes our unit test `MySQLDialectTest::quoteStringTest`